config/config.json

```diff
@@ -168,6 +175,8 @@
         "misc-virtual",
         "performance-inefficient-algorithm",
         "performance-move",
+        "performance-no-automatic-move",
+        "performance-trivially-destructible",
```

Comment:
This entry should go after `"performance-noexcept"` line to keep things in alphabetical order.